app/bundles/FormBundle/Views/Field/text.html.php

```diff
@@ -26,13 +26,13 @@
 if ('textarea' == $containerType):
 $textInput = <<<HTML
 
-                <textarea $inputAttr>{$field['defaultValue']}</textarea>
+                <textarea $inputAttr aria-label="Textarea field">{$field['defaultValue']}</textarea>
```

Comment:
```suggestion
                <textarea $inputAttr aria-label="Text area field">{$field['defaultValue']}</textarea>
```

I think these should be human readable, so it needs a space I believe.